modules/oe_paragraphs_media/oe_paragraphs_media.post_update.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/**
+ * @file
+ * Post update functions for the OE Paragraphs Media module.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Entity\Entity\EntityFormMode;
+use Drupal\Core\Config\FileStorage;
+
+/**
+ * Installs Banner paragraph.
+ */
+function oe_paragraphs_media_post_update_10001(array &$sandbox) {
```

Comment:
This is not needed anymore since enabling the oe_paragraphs_banner module will import its configuration. Also it would not belong in the oe_paragraphs_media module.
This update hook needs to enable the oe_paragraphs_media_storage_field module instead.